radosgw/README.md

```diff
@@ -15,4 +16,14 @@ For example:
 It will look for `/etc/ceph/ceph.client.admin.keyring` with which to authenticate.  You can get `ceph.client.admin.keyring` from another Ceph node.
 
 Commonly, you will want to bind-mount your host's `/etc/ceph` into the container.  For example:
-`docker run -d --net=host -e RGW_NAME=myrgw -v /etc/ceph:/etc/ceph ceph/radosgw`
+`docker run -d -e RGW_NAME=myrgw -v /etc/ceph:/etc/ceph ceph/radosgw`
+
+To enable an external CGI interface instead of civetweb set:
+
+* `RGW_REMOTE_CGI=1`
+* `RGW_REMOTE_CGI_HOST=192.168.0.1`
+* `RGW_REMOTE_CGI_PORT=9000`
+
+And run the container like this `docker run -d -e RGW_NAME=myrgw -e RGW_REMOTE_CGI=1 -e RGW_REMOTE_CGI_HOST=192.168.0.1 -e RGW_REMOTE_CGI_PORT=9000 -v /etc/ceph:/etc/ceph ceph/radosgw
```

Comment:
Well it depends which use case you're in.
This implementation doesn't support multi-backends, should we?